src/session.rs

```diff
@@ -19,29 +20,32 @@ pub struct Session {
 }
 
 impl SessionMiddleware {
-    pub fn new(cookie: &str, secure: bool) -> SessionMiddleware {
+    pub fn new(cookie: &str, key: Key, secure: bool) -> SessionMiddleware {
```

Comment:
At first i used &[u8] for key, but changed it since Key::from_master can panic if key.len() < 32. I didn't want new(..) to panic.